Resources/Prototypes/_DV/tags.yml

```diff
@@ -222,6 +222,37 @@
 - type: Tag # Ovinia, allows them to use specified emotes
   id: OviniaEmotes
 
+#reinforced cams start
+- type: Tag
+  id: cameracon
+
+- type: Tag
+  id: cameraeng
+
+- type: Tag
+  id: camerasec
+
+- type: Tag
+  id: camerasci
+
+- type: Tag
+  id: camerasup
+
+- type: Tag
+  id: cameracom
+
+- type: Tag
+  id: cameraser
+
+- type: Tag
+  id: cameramed
+
+- type: Tag
+  id: cameragen
+
+- type: Tag
+  id: cameraai
+#reinforced cams end
```

Comment:
Reason for the tags? I know they look to be used during the construction graph stuff but normal cameras don't use tags so I'm wondering if they are even needed in the first place?